### PR TITLE
Display how much time is left to AUP expiry

### PR DESCRIPTION
**Pull request by garaimanoj**

### CODE REVIEW

**Comment by garaimanoj** [comment]
When AUP validity in days is 0
image

When AUP validity in days is a positive number
image

File: iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/user/detail/user.detail.component.html [hunk 1]
```diff
@@ -83,12 +83,18 @@ <h3 class="profile-username text-center">{{$ctrl.user.name.formatted}}</h3>
                 </td>
             </tr>
             <tr ng-if="$ctrl.aupIsEnabled()">
-                <td class="text-bold text-left">
+                <td class="text-bold text-left" ng-if="!$ctrl.user.aupSignatureExpiryDate">
                     Signed AUP
                 </td>
-                <td class="text-right" ng-if="$ctrl.hasAupSignatureTime()">
+                <td class="text-bold text-left" ng-if="$ctrl.user.aupSignatureExpiryDate">
+                    AUP signature expire in
```
Comment (enricovianello) [inline iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/user/detail/user.detail.component.html:90]: typo: expires

**Review by enricovianello — CHANGES_REQUESTED** [review]

File: iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/user/detail/user.detail.component.html (hunk 1 above)
Comment (garaimanoj) [reply iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/user/detail/user.detail.component.html:90]: I wanted to say: `AUP signature will expire in`, shall I add `will`?

Comment (enricovianello) [reply iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/user/detail/user.detail.component.html:90]: i's a bit long :) We can leave it as it is or at least a shorter "AUP expiration"? what do you think?

**Review by enricovianello — APPROVED** [review]

**Pull request merged**